### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [nhungnguyen123/CodeMovie#1]
Where is your README file @nhungnguyen123?

**Comment by nhungnguyen123** [comment]
@coderschoolreview .At here
https://github.com/nhungnguyen123/CodeMovie/blob/master/CodeMovie/README.md

When I create project. I create file README inside.



On Mon, Feb 20, 2017 at 6:45 PM, coderschoolreview <notifications@github.com

**Comment by coderschoolreview** [comment]
Have you see this [link](http://courses.codepath.com/snippets/intro_to_android/readme_templates/project_1_readme.md?raw=true) at the assignment. Please read the assignment carefully!!
![README template](http://i.imgur.com/0y2e7SQ.png)

**Comment by nhungnguyen123** [comment]
I am so sorry. I try to show my image at README but it not working. I am going to ask my classmate
 to do it again  when I go to CoderSchool in evening. See you soon .
You can clicks on http://i.imgur.com/4Xh76rm.gif to see my project again . 
Thank you.

**Comment by coderschoolreview** [comment]
No, I mean you should update your README file similar to the README template, what makes our CoderSchool team easier to review. @nhungnguyen123

**Comment by nhungnguyen123** [comment]
I recently updated file README make it easier to look . I hope it will be OK.
@coderschoolreview Sorry about my README again.

**Comment by coderschoolreview** [comment]
Thank you, it looks better now.

**Comment by coderschoolreview** [comment]
Hi @nhungnguyen123, thansk for submit `assignment 1`

# Overall 

- The app runs fine without any troubles. Good work! :+1:

## Some good features

- Good job on using `Retrofit` to your app.

- Good to see you playing around with `RelativeLayout`. `RelativeLayout` is one of the most powerful responsive-first layout systems available across web and mobile platforms and provides better control in comparison to a linear layout.

- Good job using the `ViewHolder pattern` to improve performance of photos `ListView`.

- Nice to see you used the `SwipeRefreshLayout` for easy stream refreshes.

- Great work on using a `placeholder graphic` during loading.

- Good job using the `ButterKnife library annotations` for declaring handles to view objects.

## Improvements

- Try to keep your code cleaner by remove your unused code (which you comments).

- You should continue to do all of optional features. You will actually learn a lot from optional features.

- Learning to [improve UI](http://guides.codepath.com/android/Styling-UI-Screens-FAQ) is one of the most important aspects of learning about Android. It would be nice if you added proper margins, padding for your views. You may also notice the difference by formatting the text in your app. These details are technically not as challenging, but adds a lot of polish to your finished app.
